File: Assignment/Papu Sahoo/Day-11/Reverse Word.py
"""
Write a program to reverse the charecters of each word in a string 
"""
def reverse_words(input_string):
   words = input_string.split()
  reversed_words = words[::-1]
  reversed_string = " ".join(reversed_words)
    return reversed_string
input_string = "God is Omnipotent and Omniscient"
reversed_string = reverse_words(input_string)
print(reversed_string)

Output:
Omniscient and Omnipotent is God

File: Assignment/Somanath Pradhan/Day 5/ReverseWord.java
public class ReserveString { 
         public static void main(String[] args) { 
                 text = "God is Omnipotent and Omniscient"; 
                 reverse = ""; 
                 String[] words = text.split(" "); 
                 for(int i=words.length-1; i>=0; i--) { 
                         reverse = reverse + words[i]+ " "; 
                 } 
                 System.out.println("Original: "+text); 
                 System.out.println("Reverse:  "+reverse); 
     System.out.println("-----------------------------------"); 
         } 
 } 
 Output: 
       Omniscient and Omnipotent is God 
